### PR TITLE
Documentation and wording stuff!

### LINKED ISSUE #154: Fix existing documentation!
Ryan has a horrific problem with needing documentation to be as explicit as possible in an arbitrary format to his liking. This is his outlet so that tiny changes are not made compulsively in every other issue.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/about.html [hunk 1]
```diff
@@ -40,54 +40,54 @@ <h3>What are Existential Graphs?</h3>
         <p>
           Proposed by <cite><a href="https://en.wikipedia.org/wiki/Charles_Sanders_Peirce">Charles 
           Sanders Peirce</a></cite>, an <cite><a href="https://en.wikipedia.org/wiki/Existential_graph">
-          Existential Graph</a></cite> is a diagrammatic or visual representation of a logical expression. 
-          There are 3 types of Existential Graphs: Alpha Existential Graphs, Beta Existential Graphs and
-          Gamma Existential Graphs. In our application, we cover Alpha Existential Graphs.        
+          Existential Graph</a></cite> is a visual, also called diagrammatic, representation of some logical expression. 
+          There are three types of Existential Graphs: Alpha, Beta and Gamma.
```
Comment (James-Oswald) [inline src/about.html:44]:
```suggestion
          There are three systems of Existential Graphs: Alpha, Beta and Gamma.
```

File: src/about.html [hunk 2]
```diff
@@ -40,54 +40,54 @@ <h3>What are Existential Graphs?</h3>
...
+          In Peirce My Heart, we have implemented only Alpha Existential Graphs (AEGs).
```
Comment (James-Oswald) [inline src/about.html:45]: Make this sound like less of a copout, say PMH was designed to handle AEGs.

File: src/about.html [hunk 3]
```diff
@@ -40,54 +40,54 @@ <h3>What are Existential Graphs?</h3>
...
+          In Peirce My Heart, we have implemented only Alpha Existential Graphs (AEGs).        
         </p>
 
-        <h4>The parts of an Alpha Existential Graph (AEG)</h4>
-        Alpha Existential Graphs are a form of Existential Graphs that are isomorphic to <cite>
-        <a href="https://en.wikipedia.org/wiki/Propositional_calculus">Propositional Logic</a></cite>
-        Propositional Logic. Graphs of this system can be used to construct entire propositional 
-        logic statements with just 3 components: 
+        <h4>The components of an Alpha Existential Graph</h4>
+        AEGs are isomorphic to <cite>
```
Comment (James-Oswald) [inline src/about.html:49]: AEG system is isomorphic to propositional logic, AEGs themselves are a way to represent propositional formulae.

File: src/about.html [hunk 4]
```diff
@@ -40,54 +40,54 @@ <h3>What are Existential Graphs?</h3>
...
-        Anything drawn on this sheet is asserted to be <b>True</b>. Phrases, subgraphs and even the 
-        entire graph can be equated to <b>True</b> or <b>False.</b> <br/>
-        Since everything drawn on the sheet is considered <b>True</b>, placing multiple atoms on the 
-        same levels can be considered as a conjunction between the atoms. This further applies to 
-        atoms and cuts on the same level as well (Note: Asserting that multiple propositions 
-        are True all at the same time in turn holds that the conjunction of those propositions 
-        is True as well). <br/>
-        Everything contained within a cut is considered negated. As such, an empty cut represents
+        Anything drawn on The Sheet is asserted as <b>True</b>. Phrases, subgraphs and The 
+        Sheet itself may evaluate to <b>True</b> or <b>False.</b> <br/>
```
Comment (James-Oswald) [inline src/about.html:72]: The empty sheet itself always evaluates to true.

File: src/about.html [hunk 5]
```diff
@@ -40,54 +40,54 @@ <h3>What are Existential Graphs?</h3>
...
+        Since everything drawn on The Sheet is <b>True</b>, placing multiple atoms on the 
+        same level may be considered creating a conjunction between these atoms. This further applies to 
+        atoms and cuts on the same level (Note: Asserting that multiple propositions 
+        are <b>True</b> all at the same time means that the conjunction of these propositions 
+        is <b>True</b> as well). <br/>
+        Because everything contained within a cut is negated, and The Sheet corresponds to <b>True</b>, an empty cut corresponds to
```
Comment (James-Oswald) [inline src/about.html:78]: You can't start a sentence with "Because"

File: src/about.html (hunk 4 above)
Comment (RyanR712) [reply src/about.html:72]: The intention was not to say that the empty sheet might evaluate to false. Instead the intention was to say everything contained within will evaluate to true or false. Updating that now

File: src/about.html (hunk 5 above)
Comment (RyanR712) [reply src/about.html:79]: Yes I can!

**Comment by RyanR712** [comment]
I don't want to drop 5000 lines of documentation changes in mid February so I'll mark this as ready for review now and divide other documentation changes based on directories I think

**Review by James-Oswald — APPROVED** [review]

**Pull request merged**